### ISSUE TITLE
[Suggestion] VR Support

### ISSUE THREAD

**Issue by dotexe1337** [RigsOfRods/rigs-of-rods#2836]
Hi, a lot of other sims such as Assetto Corsa and iRacing have VR support now through SteamVR or some other method. It would be really nice if RoR could support this eventually. I think it would be awesome to drive the monster truck mods, doing crashes with cars, etc in VR,

**Comment by only-a-ptr** [comment]
Hi, VR support will eventually happen but don't hold your breath. The graphics framework we presently use (and are happy with), OGRE, doesn't have a built-in VR support. It's progressive spinoff OGRE-NEXT does but so far we didn't find the strength to port, as OGRE is stable and has excellent support. See also https://www.ogre3d.org/about/what-version-to-choose

**Comment by dotexe1337** [comment]
Yes, I am very well aware of OGRE. I have played RoR since around 2007 and used to use the OGRE level editor quite a bit to make my own maps back in the day.
I'm not very well versed in game development so I didn't realize that the engine needed VR support.
Well, hopefully it will happen in the future. I excitedly await that day, it will be awesome! :)
